buckets.js

```diff
@@ -27,7 +27,8 @@ define('buckets', [], function() {
             // Camel-case it.
             property = property[0].toUpperCase() + property.substr(1);
 
-            for (var i = 0, e; e = prefixes[i++];) {
+            for (var i = 0, e; i < prefixes.length; i++) {
```

Comment:
Another way to work around this jshint warning would be to wrap the assignment in brackets iirc. Longhand is probably better though :)

This means length is evaluated on every iteration that may affect perf if the loops are large. Assigning the length to a var on the left hand side would fix that.

Not sure about the var e in the loop initialisation. It looks a bit odd to me. Could the var not be set in the loop more directly?